docs/sdk/eventbus-hooks.md

```
@@ -0,0 +1,153 @@
+
+**Infrastructure (`BBT.Workflow.Infrastructure`)**
+
+- `HookedDistributedEventBus` decorates `IDistributedEventBus`
+- `EventBusHookServiceCollectionExtensions` registers the decorator
+- `EventHookServiceCollectionExtensions` provides `AddEventHook` helpers
+- Hook implementations:
+  - `InstanceSubCompletedEventHook`
+  - `InstanceSubStateChangedEventHook`
+  - `InstanceCanceledEventHook`
+
+**Integration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)